android/res/layout/onmap_downloader.xml

```diff
@@ -8,7 +8,7 @@
   android:visibility="gone"
   tools:visibility="visible">
   <LinearLayout
-    android:layout_width="wrap_content"
+    android:layout_width="212dp"
```

Comment:
Is hard-coding the width required? Will it work on smaller-screen devices?

Reply:
It's required to keep the correct size, it still works on small devices because `dp` is a relative unit :)

on a Galaxy Nexus (4.65", 720x1280):
![image](https://user-images.githubusercontent.com/26939824/199357698-747fcb26-d71b-456f-b8a4-90c61812d7af.png)
![image](https://user-images.githubusercontent.com/26939824/199357728-7ada6553-0e0d-4550-b841-5202378e3401.png)